apps/backend/src/modules/chats/chat.controller.ts

```diff
@@ -295,6 +307,34 @@ class ChatController extends BaseController {
 			status: HTTPCode.OK,
 		};
 	}
+
+	/**
+	 * @swagger
+	 * /chats/unread-count:
+	 *    get:
+	 *      tags:
+	 *        - Chats
+	 *      security:
+	 *        - bearerAuth: []
+	 *      description: Get a count of unread messages
+	 *      responses:
+	 *        200:
+	 *          description: Successful operation
+	 *          content:
+	 *            application/json:
+	 *              schema:
+	 *                type: number
+	 */
+	public async getUnreadMessageCounter(
```

Comment:
```suggestion
	public async getUnreadMessageCount(
```
Change it everywhere